### PR TITLE
run micro-benchmark for tensor instructions

### PR DESCRIPTION
**Pull request by arnej27959**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

@havardpe or @geirst please review

### CODE REVIEW

**Review by havardpe — APPROVED** [review]
I am a little worried that this will break the graph system

File: tests/performance/tensor_eval/tensor_instructions.rb [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  BM_TYPE = 'benchmark_type'
+
+  def setup
+    super
+    set_owner("arnej")
+  end
+
+  def test_instruction_benchmark
+    set_description("Test performance of low-level tensor instructions")
+    deploy_app(SearchApp.new.sd(selfdir+"foobar.sd"))
```
Comment (geirst) [inline tests/performance/tensor_eval/tensor_instructions.rb:21]: Consider renaming to `test.sd` and add the file to the PR.

File: tests/performance/tensor_eval/tensor_instructions.rb [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    searchnode = vespa.search['search'].first
+    output = searchnode.execute("#{Environment.instance.vespa_home}/bin/vespa-tensor-instructions-benchmark")
```
Comment (geirst) [inline tests/performance/tensor_eval/tensor_instructions.rb:23]: I think we need an option to the program where we run with fewer implementations and fewer benchmark cases. Currently we have ~80 benchmark cases and 6 implementations. Could also consider to graph only a subset.

File: tests/performance/tensor_eval/tensor_instructions.rb (hunk 1 above)
Comment (arnej27959) [reply tests/performance/tensor_eval/tensor_instructions.rb]: there's already a "test.sd" there, but havardpe suggested "dummy.sd" which makes it clear it's just a placeholder.

File: tests/performance/tensor_eval/tensor_instructions.rb (hunk 2 above)
Comment (arnej27959) [reply tests/performance/tensor_eval/tensor_instructions.rb:23]: yes, that's WIP in another PR...

File: tests/performance/tensor_eval/tensor_instructions.rb (hunk 1 above)
Comment (geirst) [reply tests/performance/tensor_eval/tensor_instructions.rb]: 👍

File: tests/performance/tensor_eval/tensor_instructions.rb (hunk 2 above)
Comment (geirst) [reply tests/performance/tensor_eval/tensor_instructions.rb:23]: 👍

**Pull request merged**